sbc_translation/models/translation_user.py

```diff
@@ -31,7 +31,7 @@ class TranslationUser(models.Model):
         "res.lang.compassion", domain=[("translatable", "=", True)],
         help="Utility field only used for the search view"
     )
-    search_competence_ids = fields.Many2many(
+    search_competence_id = fields.Many2many(
         "translation.competence",
```

Comment:
I think you didn't resolve the conflicts correctly. The bad changes are back.

Reply:
you're right (:

Reply:
Still not resolved.

Reply:
you're right (: